components/css/index.css

```diff
@@ -0,0 +1,149 @@
+@import 'normalize.css';
```

Comment:
Why is this moved out of normalize?

Reply:
Left over from original direction. I'll remove that.